week-5-ci-and-production/w05e02-continuous-deployment/.github/workflows/api.yml

```diff
@@ -0,0 +1,38 @@
+name: API – Stage Deploy
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'shopping-lister-api/**'
+
+jobs:
+  test_build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Cache node modules
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Install dependencies
+        run: npm ci
+        working-directory: ./shopping-lister-api
+      - name: Format code and check for errors
+        run: npm run lint:full
+        working-directory: ./shopping-lister-api
+      - name: Run tests
```

Comment:
Możesz dodać jeszcze jeden krok:

_Run e2e tests_

i dodać tam:

`npm run test:e2e`

one powinny działać (tam chyba `.skip` dałem, ale to się wykonuje z `exit code 0` - więc przejdzie)

Reply:
Dodałem krok do testów e2e 👍